app/2.0/docs/devguide/style-shadow-dom.md

```diff
@@ -515,6 +515,64 @@ class myElement extends Polymer.Element(){
 }
 ```
 
+### Style directional text using the :dir() selector
+
+The `:dir()` CSS selector allows for styling text specific to its orientation 
+(right-to-left or left-to-right). See the [documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) for more information on the :dir() 
+selector.
+
+For elements that extend `Polymer.Element`, add the `Polymer.DirMixin` mixin to use
+`:dir()` styling.
+
+`index.html` { .caption}
```

Comment:
Taste issue, but I'd be inclined to put the element example first (you just introduced the mixin, and the element demonstrates its use). 

The index example shows how you use the element.

Reply:
... And how you set `dir` on the document in markup.